src/views/AmalgamationOut.vue

```diff
@@ -118,6 +118,9 @@
                   ref="effectiveDateRef"
                   :class="{ 'invalid-component': !effectiveDateValid && showErrors }"
                   class="pt-6 px-4"
+                  :effectiveDateLabel="'Effective Date of Amalgamation Out'"
+                  :dateLabel="'Date of Amalgamation Out'"
+                  :dateRequiredMsg="'A Date of Amalgamation Out is required'"
```

Comment:
Another thing that should work is a static binding.

Eg, `effectiveDateLabel="Effective Date of Amalgamation Out"`